client/README.md

```diff
@@ -1,4 +1,9 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Sky9
+
+## Prerequisites
+    - NodeJS
+To create a project with React, run:
```

Comment:
This is not something you would typically do after checking out this repo, right?

Reply:
Yes that is correct, i will remove it.